app/models/user.js

```diff
@@ -0,0 +1,43 @@
+import DS from 'ember-data';
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'DS' is not defined.<br>Too many errors. (2% scanned).